pandas/tests/test_multilevel.py

```diff
@@ -1711,6 +1711,14 @@ def test_repeat(self):
         m_df = Series(data, index=m_idx)
         assert m_df.repeat(3).shape == (3 * len(data), )
 
+    def test_in(self):
```

Comment:
can you call this test_contains and move next to pandas/tests/indexing/test_multiindex.py